api/cluster/resource/templater_test.go

```diff
@@ -37,6 +37,11 @@ import (
 	"github.com/gojek/merlin/models"
 )
 
+const (
+	environmentName  = "dev"
```

Comment:
Nit: Shall we prefix these names with `test` so it doesn't somehow interfere with other variables scoped under the `resource` package? Alternatively (and preferably), this package can be named `resource_test` and focus on only testing the public API - but it seems that the tests are currently closely coupled with the private functions, so this can be a future improvement. (The same comment applies to some other test files in this PR too.)

Reply:
> Shall we prefix these names with test so it doesn't somehow interfere with other variables scoped under the `resource` package?
Ahh good point, I'll do this then; but yes, seems like separating the packages will be a good idea in the long run.